tests/unit_tests/test_geometry.py

```diff
@@ -6,6 +6,49 @@
 import pytest
 
 
+def test_contains_cell():
+
+    c1 = openmc.Cell()
+    c2 = openmc.Cell()
+    c3 = openmc.Cell()
+    c3 = openmc.Cell()
+    c4 = openmc.Cell()
+
+    
+    geom1= openmc.Geometry([c1, c2, c3])
+
+    assert (c1 in geom1)
+    assert (c2 in geom1)
+    assert (c3 in geom1)
+    assert not (c4 in geom1)
+
+    
+    univ1 = openmc.Universe(name='univ1', cells=[c1, c2, c3])
+    geom2 = openmc.Geometry(univ1)
+
+    assert (c1 in geom2) 
+    assert (c2 in geom2)
+    assert (c3 in geom2)
+    assert not (c4 in geom2)
+
+
+    c5 = openmc.Cell()
+    c6 = openmc.Cell()
+    univ2 = openmc.Universe(name='univ2', cells=[c5, c6])
+    c7 = openmc.Cell(fill=univ2)
+    c8 = openmc.Cell(fill=univ1)
+    univ_comp = openmc.Universe(name='composed universe',cells=[c7, c8])
+    geom_comp = openmc.Geometry(univ_comp)
+
+    assert c5 in geom_comp
+    assert c6 in geom_comp
+    assert c7 in geom_comp
+    assert c8 in geom_comp
+    assert c1 in geom_comp
+
+
+
```

Comment:
A few thoughts:

1. I don't think this is quite PEP8, but I don't have the style off the top of my head.
2. Line 27 could just be `geom1.root_universe = univ1` but isn't essential. 
3. The test suite looks really good for having wide coverage of all possible scenarios.
4. There is some repeated code, I can't decide if the effort of making it DRY is worth it here though. It is a test suite after all.